processor/k8sattributesprocessor/e2e_test.go

```diff
@@ -719,6 +720,146 @@ func TestE2E_MixRBAC(t *testing.T) {
 	}
 }
 
+// Test with `filter::namespace` set and only role binding to collector's SA. We can't get node and namespace labels/annotations.
+// While `k8s.pod.ip` is not set in `k8sattributes:extract:metadata` and the `pod_association` is not `connection`
+// we expect that the `k8s.pod.ip` metadata is not added.
+func TestE2E_NamespacedRBACNoPodIP(t *testing.T) {
+	testDir := filepath.Join("testdata", "e2e", "namespacedrbacnopodip")
```

Comment:
is it possible rename directory from namespacedrbacnopodip to `namespaced_rbac_no_pod_ip`, although existing  e2e test directories' names are also using similar way of naming, but in this case I thinks `namespacedrbacnopodip` is too hard to understand for others.